lib/fastlane/plugin/revenuecat_internal/helper/github_helper.rb

```diff
@@ -8,7 +8,7 @@ module Fastlane
 
   module Helper
     class GitHubHelper
-      SUPPORTED_PR_LABELS = %w[breaking build ci docs feat fix perf refactor style test next_release dependencies phc_dependencies minor].to_set
+      SUPPORTED_PR_LABELS = %w[breaking build ci docs feat fix perf RevenueCatUI refactor style test next_release dependencies phc_dependencies minor].to_set
```

Comment:
Hmm should we make it lowercase? Like `revenuecatui`. Since all the others are lowercase

Reply:
Done, and I renamed the label on the iOS repo.